File: core/Core/Models/Atm.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class Atm
{
    public int AtmId { get; set; }

    public int? BranchId { get; set; }

    public required string Location { get; set; }

    public required string Status { get; set; }

    public virtual Branch? Branch { get; set; }
}

File: analytic/domain/atm/atm.py
from datetime import datetime


class Atm:
    def __init__(self,
                 atm_id: int,
                 location: str,
                 status: str,
                 branch_id: int):
        self.atm_id: int = atm_id
        self.location: str = location
        self.status: str = status
        self.branch_id = branch_id